mopidy/config/__init__.py

```diff
@@ -182,6 +183,12 @@ def _validate(raw_config, schemas):
             errors[schema.name] = error
         if result:
             config[schema.name] = result
+        if schema.name in sections:
+            sections.remove(schema.name)
+
+    for section in sections:
+        logger.debug('Ignoring unknown config section: %s', section)
```

Comment:
Do we need the noise / red herrings? My other thought was to propagate this up the chain in errors or as an extra return value, but this is just the quick shortcut to at least make it a bit discoverable what is going on. I've also dropped the idea of having a config to disable this.